packages/host/app/services/operator-mode-state-service.ts

```diff
@@ -532,4 +532,17 @@ export default class OperatorModeStateService extends Service {
       },
     }));
   });
+
+  async openCardInInteractMode(card: CardDef) {
+    this.clearStacks();
+    let newItem = new StackItem({
+      card,
+      stackIndex: 0,
+      owner: this, // We need to think for better owner
```

Comment:
I see `this` is used for `owner` in various places and is apparently undesirable… I’m not sure whether this is the owner concept of Ember but if it is, could we use `getOwner(this)`?

Reply:
Yes it is the owner concept of ember. We need the `owner` arg here because inside `StackItem`, we'll create a `CardResource` and we need to define the owner of that resource. The problem here is the owner will be the operator mode state service, and that resource will live with the service. I guess ideally the owner must be the interact submode component but when we instantiate the newStackItem the component hasn't existed yet.